playbooks/02 - Use Case - CICD/Setup CICD Environment.json

```diff
@@ -321,6 +367,16 @@
             "group": null,
             "uuid": "7bf0fb04-71c5-4d10-9f2f-c3e43b98dfea"
         },
+        {
+            "@type": "WorkflowRoute",
+            "name": "Create CICD Env Global Variable -> Get Investigate Outbreak Alerts Schedule Details",
```

Comment:
Investigate Outbreak Alert --> do we need this here or its by mistake?

Reply:
these are copied steps and in link it shows at is and not impacted the playbook, but we should need to file the bug for same